src/test/java/com/wikia/webdriver/common/contentpatterns/MercurySubpages.java

```diff
@@ -31,16 +31,16 @@ private MercurySubpages() {}
   public static final String CC_MAIN_PAGE = "Mercury_CC_Wikia";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)